helm_deploy/prisoner-offender-search/snapshot.sh

```diff
@@ -1,50 +1,48 @@
 #!/bin/bash
 set -e
-HTTPIE_SESSION="$HOME/.httpie_session.json"
```

Comment:
since we're not logging in to elastic search there is no point having a session file.  The contents are always empty.